scripts/restore.sh.in

```diff
@@ -270,13 +291,22 @@ highstate_bootstrap() {
         pillar="{'orchestrate': {'node_name': '$bootstrap_id'}}"
 }
 
+reconfigure_k8s_obj() {
+    SALT_MASTER_CALL=(crictl exec -i "$(get_salt_container)")
+
+    "${SALT_MASTER_CALL[@]}" salt-run --state-output=mixed state.orchestrate \
+        metalk8s.addons.nginx-ingress-control-plane.deployed \
```

Comment:
Maybe we could simply run `metalk8s.deployed` here? Being idempotent and only executed on the master, I'm thinking it's some kind of "K8s highstate"... If you think so, we could do with a debt ticket to fix in 2.5 :)

Reply:
Since for the moment we basically replace every object I think it's not a good idea because we will lose all tuning done by the user on k8s objects when restoring which is unexpected IMO

Reply:
Good point, let's wait for #1996 then